### ISSUE TITLE
Another one who can't unlock wallet with correct password

### ISSUE THREAD

**Issue by mredde** [Multibit-Legacy/multibit-hd#1012]
It seems I'm another victim of the what seem to be becoming the 'dreaded' Multibit HD (0.5.1) bug i.e. Absolutely 100% correct password won't unlock wallet containing several bitcoins. After attempting a recovery using wallet words I see the bitcoin balance in the background, but when I click 'Finish', I getthe password unlock error. I even set the simplest password posisble - '1' ! 

The solutions below look excellent - and it makes sense that it's something to do with recent txn's - but I'm having some trouble running them on my MacOSX Sierra machine. I'll have another play. 

(I have a 24 word seed btw, so can't restore into alternative wallet solutions such as Breadwallet, Mycelium)

In the meantime, I've logged a support ticket with MB support - we'll see if they provide an ETA of an updated release that fixes whatever the bug is. This is a big problem and getting bigger.

Fingers crossed, and any assistance would be massively appreciated - by MB support or the community ...

Thanks in advance

**Comment by noirex999** [comment]
any news from your side? I just got an answer from support but they said to use breadwallet/simple bitcoin wallet to recover, i guess they didn't read my post, so i have pasted it into my reply

**Comment by mredde** [comment]
Hey,

Yes, I got a reply. As follows;

------------------------
"A known bug in Multibit is causing the error you are experiencing. Your funds remain safe and recoverable.

Please recover your wallet with using Breadwallet or Simple Bitcoin wallet. They are both mobile wallets that work great with Multibit wallet word recovery. Breadwallet (https://breadwallet.com/) is available on both iPhones and Androids and Simple Bitcoin wallet (https://play.google.com/store/apps/details?id=com.btcontract.wallet&hl=en) is available on Androids.

A desktop recovery method using Electrum will be released soon. 

We are currently working on other solutions for this issue and will advise when this is available."

------------------------
The problem is I'm an iPhone user and Breadwallet doesn't support 24 word seeds. I don't know if Simple Wallet does, if so, then I'd get a cheap Android device. Anyone know?

I've gone back to Multibit and asked whether there's an ETA for the fix... I'll keep you posted if I get anything else back.

**Comment by noirex999** [comment]
Yea its the same email i got from support, they didnt look at all the detailed information i gave in my post then. Anyway don't waste money on android device because i have already tried both with Breadwallet and Simple Bitcoin Wallet and my balance is still 0. This procedure wont solve the issue, we need another kind of fix. I am thinking that they will need our backup files with some of the possible wallet's passwords and seeds

**Comment by mredde** [comment]
Latest email from Multibit support. I'll see what I can find on a potential new release of Electrum.

Multibit are going to lose a lot of users over this ...

--------------------

It does not. [Question whether Simple Wallet supported 24 word seeds]

We have not found the root cause of this bug in Multibit, ETA is unknown. However the support in Electrum is ready, they are making other changes to the wallet before releasing. I am hoping the release will be made within a week.

Another option is to derive your keys from your wallet words and then importing these in the existing version of Electrum.

Here is how: 
 
First, we will need to generate your private keys out of your wallet words. Use this tool here: https://iancoleman.github.io/bip39/

Enter your wallet words in the BIP39 Mnemonic field.
 
 

Next, type m/0'  in the BIP32 Derivation Path under the BIP32 tab. A BIP32 Extended Private Key will be generated. This is what we will need to recover in Electrum.

image

Now we are ready to recover your wallet in Electrum. Download the application here: https://electrum.org/#download
In the Install Wizard select Restore a wallet or import keys and Standard Wallet. Click Next.
Use public/private key

image
 


Next, paste the BIP32 Extended Private Key in the seed field. Select Next and your wallet will be created in the Electrum client. This may take a moment. 

 
image

**Comment by jimmykl** [comment]
Yes, what a joke. Their app is completely broken, they cannot offer an ETA and only solution is use another wallet. With so many other offerings out there I'll never bother trying you again Multibit!

They should really own up to the problem and disable downloads on their site too.

**Comment by eira123** [comment]
thanks it werkt

**Comment by protemson1** [comment]
@mredde 
@noirex999 
multibit bitcoin wallet has many bugs and issues, and I have solved most of its issues, I can recover your stuck transaction, and lost password for multibit bitcoin wallet file, before, I have recovered several multibit bitcoin wallets for my customers, you can find me at : walletsrecoveryservice@gmail.com

**Comment by doge2021** [comment]
> It seems I'm another victim of the what seem to be becoming the 'dreaded' Multibit HD (0.5.1) bug i.e. Absolutely 100% correct password won't unlock wallet containing several bitcoins. After attempting a recovery using wallet words I see the bitcoin balance in the background, but when I click 'Finish', I getthe password unlock error. I even set the simplest password posisble - '1' !
> 
> The solutions below look excellent - and it makes sense that it's something to do with recent txn's - but I'm having some trouble running them on my MacOSX Sierra machine. I'll have another play.
> 
> (I have a 24 word seed btw, so can't restore into alternative wallet solutions such as Breadwallet, Mycelium)
> 
> In the meantime, I've logged a support ticket with MB support - we'll see if they provide an ETA of an updated release that fixes whatever the bug is. This is a big problem and getting bigger.
> 
> Fingers crossed, and any assistance would be massively appreciated - by MB support or the community ...
> 
> Thanks in advance

do you recovered it. if not. your seed maybe has bugs. you can check the post [multibit HD old seed problem](https://btc2doge.com/tag/multibit-hd-old-wallet-restore/)

**Comment by doge2021** [comment]
> any news from your side? I just got an answer from support but they said to use breadwallet/simple bitcoin wallet to recover, i guess they didn't read my post, so i have pasted it into my reply

this is caused by the old multibit HD wallet.
https://btc2doge.com/multibit-hd-wallet-seed-words-not-work/

**Comment by doge2021** [comment]
> any news from your side? I just got an answer from support but they said to use breadwallet/simple bitcoin wallet to recover, i guess they didn't read my post, so i have pasted it into my reply

@mredde @noirex999 
the old multibit HD wallet has bug, so you cant use them on other wallets
https://btc2doge.com/multibit-hd-wallet-seed-words-not-work/
